### PR TITLE
Fix issue with leaving queue

### PR DESCRIPTION
**Pull request by poszer**
- Fix issue with leaving queue, Already tested and confirmed as fixed, comment here : 

Closes: https://github.com/azerothcore/mod-1v1-arena/issues/2

https://github.com/azerothcore/mod-1v1-arena/issues/3#issuecomment-519862790

### CODE REVIEW

File: src/npc_arena1v1.cpp [hunk 1]
```diff
@@ -279,8 +279,12 @@ class npc_1v1arena : public CreatureScript
 
         case 3: // Leave Queue
         {
+            uint8 arenaType = ARENA_TYPE_5v5;
+            if (player->InBattlegroundQueueForBattlegroundQueueType(BATTLEGROUND_QUEUE_5v5))
+                arenaType = ARENA_TYPE_5v5;
```
Comment (Helias) [inline src/npc_arena1v1.cpp]: why this reassignment to the same value?

**Comment by Helias** [comment]
This PR contain code duplicate of https://github.com/azerothcore/mod-1v1-arena/pull/4/ 

immagine

I think we could close it and focus on https://github.com/azerothcore/mod-1v1-arena/pull/4/ that contains more improvements, do you agree?
@poszer,  @Winfidonarleyan?

File: src/npc_arena1v1.cpp (hunk 1 above)
Comment (poszer) [reply src/npc_arena1v1.cpp]: > This PR contain code duplicate of #4
> 
> immagine
> 
> I think we could close it and focus on #4 that contains more improvements, do you agree?
> @poszer, @Winfidonarleyan?

Are you serious? :D

**Comment by poszer** [comment]
> This PR contain code duplicate of #4
> 
> immagine
> 
> I think we could close this and focus on #4 that contains more improvements, do you agree?
> @poszer, @Winfidonarleyan?

Screenshot_2

**Comment by poszer** [comment]
> This PR contain code duplicate of #4
> 
> immagine
> 
> I think we could close this and focus on #4 that contains more improvements, do you agree?
> @poszer, @Winfidonarleyan?

And to be clear, this PR is created before other one

**Comment by masterking32** [comment]
tested and working. I can correctly leave the Arena Queue clicking on NPC gossip

**Comment by Helias** [comment]
We could merge both PRs to give the credits to both.

**Comment by FrancescoBorzi** [comment]
agree, please merge both

**Comment by milestorme** [comment]
Thank you @poszer can merge this

**Comment by FrancescoBorzi** [comment]
for some reasons squash failed, it merged without squashing commits

**Comment by poszer** [comment]
> for some reasons squash failed, it merged without squashing commits

The other PR is not needed. The module works now perfectly except Skirmish. And other PR commit wont fix it

**Comment by FrancescoBorzi** [comment]
@Poszer please discuss it in the other pr, not here. I don't have time to check it now

**Pull request merged**